composer.json

```diff
@@ -21,7 +21,8 @@
 
 	"require-dev": {
 		"satooshi/php-coveralls": "dev-master",
-		"phpunit/php-file-iterator": "1.3.3"
+		"phpunit/php-file-iterator": "1.3.3",
+		"phpunit/phpunit": "~4.0"
```

Comment:
Just so we're clear, as one of the other maintainers, I'm pro phpunit as a composer dependency, but I think @cordoval and @aik099 outnumber me two to one :(